registry.json

```diff
@@ -117,6 +117,21 @@
       "telegram":"CardanoPoolU",
       "operatorName":"RALPH",
       "github":""
+   },
+      {
+      "ticker":"LGLP",
+      "poolId":"6936fe0511eada38cbc229ce57dda32b8db2a27e237689b46afa4228",
+      "social":{
+         "facebook":"",
+         "twitter":"LglpStudios",
+         "website":"https://twitter.com/LglpStudios",
+         "reddit":"",
+         "instagram":"",
+         "youtube":""
+      },
+      "telegram":"LGLP",
```

Comment:
I can't see this user on TG. It must be the ID of a user and not a group or channel

Reply:
I messed up and had thought i'd created the user id for the account - changing TG value to be the actual user id.